proxy/fluentd/test/plugin/test_parser_openlineage.rb

```diff
@@ -1,141 +0,0 @@
-require "helper"
-require "fluent/plugin/parser_openlineage.rb"
-
-class OpenlineageParserTest < Test::Unit::TestCase
```

Comment:
What's happening with those tests? They're important.

Reply:
Sorry, I had mistakenly removed them when I was re-organizing the folders to generate the plugin. I've included them back in.